packages/foreman/rubygem-eventmachine/rubygem-eventmachine.spec

```diff
@@ -1,30 +1,27 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name eventmachine
 
-# This enables to run full test suite, where network connection is available.
-# However, it must be disabled for Koji build.
-%{!?network: %global network 0}
-
-Summary:    Ruby/EventMachine library
-Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    1.0.6
-Release:    3%{?dist}
-Group:      Development/Languages
-License:    GPLv2 or Ruby
-URL:        http://rubyeventmachine.com
-Source0:    http://gems.rubyforge.org/gems/%{gem_name}-%{version}.gem
-Requires:   %{?scl_prefix_ruby}ruby(rubygems)
-Requires:   %{?scl_prefix_ruby}ruby(release)
-Provides:   %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.6
+Release: 4%{?dist}
+Summary: Ruby/EventMachine library
+Group: Development/Languages
+License: Ruby and GPL
+URL: http://rubyeventmachine.com
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby 
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix_ruby}ruby-devel
-BuildRequires: %{?scl_prefix_ruby}rubygem(test-unit)
-# Enables SSL support.
-BuildRequires: openssl-devel
```

Comment:
Looks like we should keep this but put it after line 24 so that it doesnt get erased in the future.